resources/lib/composite_addon/plex/plexserver.py

```diff
@@ -239,8 +239,9 @@ def connection_test(self, tag, uri):
         url_parts = urlparse(uri)
         status_code = requests.codes.not_found  # pylint: disable=no-member
         try:
+            verify_cert = True if uri.startswith('https') else False
```

Comment:
I think this should be `verify_cert = uri.startswith('https') and SETTINGS.get_setting('verify_cert')` so the `Certificate Verification` setting is honored.
Can you see if that still works in your case